packages/terra-clinical-data-grid/src/subcomponents/Cell.module.scss

```diff
@@ -39,9 +41,12 @@
         box-shadow: var(--terra-clinical-data-grid-cell-selected-focus-box-shadow, inset 0 0 0 1px #0092e0, inset 0 0 0 4px rgba(76, 178, 233, 0.3));
         outline: none;
       }
-
-      &:hover {
-        background-color: var(--terra-clinical-data-grid-cell-selected-hover-background-color, #b3ddf7);
+      
+      /* stylelint-disable max-nesting-depth */
```

Comment:
can we disable next line or does this need to be re-enabled?

Reply:
Re-enabled 9225f6d